auth_backends/urls.py

```diff
@@ -2,15 +2,16 @@
 
 Add these to your project's `urlpatterns` to avoid duplicating code.
 """
-from django.conf.urls import url, include
+from django.conf.urls import include
+from django.urls import re_path
```

Comment:
`include` is also now available via `django.urls`, so this whole thing can be simplified to `from django.urls import include, re_path`.

Reply:
sounds good, will change